parts/21-mutate.sh

```diff
@@ -518,3 +518,24 @@ mutate_fail-wfi() { ## <wf-invocation-d> [--commit]: Sets a workflow invocation
 	commit=$(should_commit "$2")
 	QUERY="BEGIN TRANSACTION; $QUERY; $commit"
 }
+
+mutate_oidc-by-emails() { ## <email_from> <email_to> [--commit]: Reassign OIDC account between users.
+	handle_help "$@" <<-EOF
+		Workaround for users creating a new account by clicking the OIDC button, with case mismatching between existing accounts. 
+		Please note that this function is case-sensitive. Fixes https://github.com/galaxyproject/galaxy/issues/9981.
+	EOF
+
+	assert_count_ge $# 2 "Must supply an email_from and an email_to";
+
+	read -r -d '' QUERY <<-EOF
+		UPDATE oidc_user_authnz_tokens
+		SET user_id=correctuser.id
+		FROM (
+			SELECT id FROM galaxy_user WHERE email='$2'
+		) AS correctuser
```

Comment:
```suggestion
		SET user_id=(SELECT id FROM galaxy_user WHERE email='$2')
```

this would work too, right?